switchboard/base.py

```diff
@@ -20,13 +21,17 @@
 
 
 class CachedDict(threading.local):
-    def __init__(self, cache=None, timeout=30):
+    def __init__(self, timeout=30):
```

Comment:
There are a lot of test failures regarding `TypeError: __init__() got an unexpected keyword argument 'cache'` now.